src/utils/server/ipfs/findProjectMetadata.ts

```diff
@@ -20,10 +19,7 @@ export const findProjectMetadata = async ({
   while (true) {
     try {
       const response = await limiter.schedule(
-        async () =>
-          await ipfsGetWithFallback<AnyProjectMetadata>(metadataCid, {
-            fallbackHostname: PUBLIC_PINATA_GATEWAY_HOSTNAME,
```

Comment:
This is the change that fixes Blunt projects: see http://localhost:3000/v2/p/505.

Fixes it because their metadata uses IPFS V1 content hashes, and pinata only supports v0 afaik: https://docs.ipfs.tech/concepts/content-addressing/#cid-conversion